routes/index.js

```diff
@@ -1,7 +1,7 @@
 const express = require( "express" );
 const router = express.Router();
 
-router.get( "/", ( req, res ) => res.render( "landing" ) );
+router.get( "/", ( req, res ) => res.render( "landing", { title: "landing-page"} ) );
```

Comment:
No, more like "What to do?", the user does not care that its the landing page.

Reply:
by changing the heading to "What to do", the tab-text will read like "What to do? | What to do?" which will just be redundand information. That is why I didn't give it an title to begin with. Would an tilte like "welcome" be better?

Reply:
"What to do? | What to do?" is fine, though.

Reply:
Again, it's not about the individual page, but the website as a whole. imo having a consistent title naming across all pages is more important than reducing some "redundant" info at points.